src/index.html

```diff
@@ -17,70 +17,71 @@
-                Home
-              </a>
-            </li>
-            <li class="left">
-              <a href="https://github.com/Stephenich">
-                GitHub
-              </a>
-            </li>
+          <li class="item">
+            <a href="https://stephenich.github.io/"> Home </a>
+          </li>
+          <li class="item">
+            <a href="https://github.com/Stephenich"> GitHub </a>
+          </li>
         </ul>
-      </nav> 
+      </nav>
     </header>
 
     <article>
-      <h2>CSS Layouts</h2>
-      <p class="text">
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde laudantium
-        distinctio temporibus iusto consequatur pariatur aut reiciendis
-        obcaecati voluptatum sint voluptatibus, debitis tempore consequuntur cum
-        quidem nihil dolorum odit natus. Ipsum, ipsa nam. Dignissimos ullam
-        optio nemo illum cupiditate nulla repudiandae ex ut. Optio veritatis
-        dicta tempora tempore impedit dolor, repellendus dolores quod
-        perferendis quibusdam at sequi quia ullam beatae. Iusto assumenda neque
-        deserunt repudiandae aspernatur quibusdam minus tenetur, earum dolorum
-        fugit odit illo maxime delectus! Quibusdam adipisci vel repellendus
-        facilis itaque. Magnam veritatis corporis fuga in omnis debitis ipsa!
-      </p>
-      <p class="text">
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem
-        dignissimos aperiam, ratione molestiae sunt odit accusantium et
-        asperiores? Nam inventore aliquam nulla culpa. Praesentium ut nam at
-        corrupti non soluta! Modi dolore, omnis sed quia quam laborum sint
-        similique mollitia nam esse, est sapiente fugiat quas placeat, tempore
-        recusandae necessitatibus harum! Perferendis saepe hic accusantium
-        voluptatem sed! Dolores, quasi voluptates? Fugiat a autem aspernatur
-        cumque beatae, itaque provident laboriosam eaque eos esse consectetur
-        quis quia eius qui reiciendis, earum possimus dignissimos deleniti quo
-        dolorem in modi velit voluptatem temporibus. Saepe! Exercitationem
-        cumque, deleniti id quidem sit nulla? Ipsum nisi reprehenderit tenetur
-        est laudantium eligendi neque repudiandae quisquam, ut maiores
-        consectetur, ex qui corrupti ab, suscipit aliquid! Est illum sint
-        distinctio? Sit assumenda et maiores culpa commodi eum delectus beatae,
-        nostrum cum ad ut! Neque tempore eius esse temporibus sunt quis aperiam.
-        Quo, quae? Ipsum pariatur, a tempora itaque explicabo similique! Ad aut
-        commodi ullam dignissimos quia maiores ratione sequi vitae doloribus
-        animi necessitatibus nesciunt numquam doloremque, qui, quos dolore sit?
-        Asperiores possimus porro dignissimos esse et odio nesciunt dolorem
-        autem!
+      <h2 class="h2">CSS Layouts</h2>
+      <div class="text">
+        <p class="text1">
```

Comment:
instead of declaring a class for each <p> i would cascade the class from the parent css class
```
.text p
```

Reply:
This makes sense, I didn't do it this time as I didn't need to. I'll keep it in mind for the future.